src/Microsoft.Azure.SignalR.Emulator/Controllers/SignalRServiceEmulatorWebApi.cs

```diff
@@ -26,10 +24,10 @@ internal class SignalRServiceEmulatorWebApi : SignalRServiceWebApiDefinition
     {
         private const string HubPattern = "^[A-Za-z][A-Za-z0-9_`,.[\\]]{0,127}$";
         private const string GroupPattern = "^\\S{1,1024}$";
-        private readonly DynamicHubContextStore _store;
+        private readonly IDynamicHubContextStore _store;
```

Comment:
As you changed to use interface, you should change every usage of this concreate class to interface. As well as `AddSignalREmulator`